etna/transforms/datetime_flags.py

```diff
@@ -229,6 +237,12 @@ def _get_year(timestamp_series: pd.Series) -> np.array:
         """Generate an array with the week number in the year."""
         return timestamp_series.apply(lambda x: x.year).values
 
+    @staticmethod
+    def _get_weekends(timestamp_series: pd.Series) -> np.array:
+        """Generate an array with the weekends flags."""
+        weekend_days = (5, 6)
+        return timestamp_series.apply(lambda x: x.weekday() in weekend_days).values
```

Comment:
Could we use something like [this](https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.weekday.html) instead of apply?

Reply:
I use it to obtain the day number  in week here `x.weekday()`

Reply:
Pandas doesn't have a special method for weekends